lab-kyle/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
This is good, I've made one comment below which will make it exactly as specified in the assignment document.